save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/utils/LocalDateTimeUtils.kt

```diff
@@ -40,6 +44,21 @@ private object JLocalDateTimeKSerializer : KSerializer<JLocalDateTime> {
     override fun serialize(encoder: Encoder, value: JLocalDateTime) = kotlinxSerializer.serialize(encoder, value.toKotlinLocalDateTime())
 }
 
+private object KDurationJsonSerializer : JsonSerializer<KDuration>() {
+    override fun serialize(value: KDuration?, gen: JsonGenerator?, serializers: SerializerProvider?) {
+        gen?.codec?.writeValue(gen, value?.toJavaDuration())
+    }
+}
+
+private object KDurationJsonDeserializer : JsonDeserializer<KDuration>() {
+    @Suppress("UnsafeCallOnNullableType")
+    override fun deserialize(parser: JsonParser?, ctxt: DeserializationContext?): KDuration =
+            parser
+                ?.codec
+                ?.readValue(parser, JDuration::class.java)
+                ?.toKotlinDuration()!!
```

Comment:
we use `requireNotNull()` in `KLocalDateTimeJsonDeserializer`.
can you please change the `KLocalDateTimeJsonDeserializer` to use the same approach on handling nulls in impossible situations

Reply:
Addressed with 7456eb016e680a806126a4a5fd870482f40fc670.